hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/configuration/FlinkOptions.java

```diff
@@ -487,6 +487,13 @@ private FlinkOptions() {
       .defaultValue(true) // default true for MOR write
       .withDescription("Schedule the compaction plan, enabled by default for MOR");
 
+  public static final ConfigOption<Integer> WRITE_BULK_INSERT_MAX_PARALLELISM = ConfigOptions
+          .key("write.bulk_insert.max.parallelism")
+          .intType()
+          .defaultValue(StreamGraphGenerator.DEFAULT_LOWER_BOUND_MAX_PARALLELISM)
+          .withDescription("The maximum number of flink write tasks, since the original " +
```

Comment:
How about we just increase the `StreamGraphGenerator.DEFAULT_LOWER_BOUND_MAX_PARALLELISM` so we can avoid the option ?

Reply:
Yep, I was thinking about it at first sight, but after referring to `StreamGraphGenerator.DEFAULT_LOWER_BOUND_MAX_PARALLELISM`, it seems that the number can not be changed since it comes from `org.apache.flink.runtime.state.KeyGroupRangeAssignment.DEFAULT_LOWER_BOUND_MAX_PARALLELISM`, which was hard coded to 128.

Reply:
What i mean is in `Pipelines` line 139, we hard code a greater value.

Reply:
i see, how about `org.apache.flink.runtime.state.KeyGroupRangeAssignment.UPPER_BOUND_MAX_PARALLELISM`, which is hardcoded to 32768? i guess it will work for almost all the workloads.